crates/octez/src/client.rs

```diff
@@ -279,6 +279,8 @@ impl OctezClient {
             "--burn-cap",
             "999",
             "--force",
+            "--whitelist",
+            format!("[\"{}\"]", account).as_str(),
```

Comment:
Does this affect the local sandbox?

Reply:
good point, we really need some tests :D I've tried before starting the sandbox which is ok, but apparently deploying a contract causes Timeout waiting for operation receipt. I'll look into it

Reply:
it doesn't actually affect it.